test.js

```diff
@@ -1,31 +1,32 @@
 import test from 'ava';
 const friendlyWords = require('./index');
+const lists = ["predicates","objects","teams","collections"];
 
```

Comment:
This is good!
Another way to save a little typing is to store the word lists directly.  For example something like:
`const lists = ["predicates","objects",...].map(category => friendlyWords[category])`

This would allow you to stop worrying about the array dereferencing in the tests, since you could now do this:
```
lists.forEach((words) => {
    t.deepEqual(words, Array.from(new Set(words))
  });
```
etc.